test/sequential/test-http-pipeline-flood.js

```diff
@@ -72,20 +88,13 @@ function child() {
   req = new Array(10241).join(req);
 
   conn.on('connect', function() {
+    //kill child after flooding
+    setTimeout(function() { conn.destroy(); }, common.platformTimeout(1000));
```

Comment:
Modified test to use `child.kill()` instead of `setTimeout()` but now it passes in v0.10.20 which does not have the flood protection mechanism. Clearly, I'm Doing Something Wrong. Here's a gist of the modified test with ES6-isms and other things that don't work on 0.10.20 taken out.  https://gist.github.com/Trott/cfe8d6d0219e1e06d416